addons/calendar/data/mail_template_data.xml

```diff
@@ -460,62 +444,57 @@
-                                    </div>
-                                    <div t-attf-style="font-size: 48px; min-height: auto; font-weight: bold; text-align: center; color: #5F5F5F; background-color: #F8F8F8; border: 1px solid {{object.user_id.company_id.email_secondary_color or '#875A7B'}};">
-                                        <t t-out="format_datetime(dt=object.start, tz=mail_tz if not object.allday else None, dt_format='d', lang_code=object.env.lang)">4</t>
-                                    </div>
-                                    <div t-attf-style="font-size: 12px; text-align: center; font-weight: bold; color: {{object.user_id.company_id.email_primary_color or '#FFFFFF'}}; background-color: {{object.user_id.company_id.email_secondary_color or '#875A7B'}};">
-                                        <t t-out="format_datetime(dt=object.start, tz=mail_tz if not object.allday else None, dt_format='MMMM y', lang_code=object.env.lang)">May 2021</t>
-                                    </div>
-                                    <div t-attf-style="border-collapse: separate; color: #5F5F5F; text-align: center; font-size: 12px; border-bottom-end-radius: 3px; font-weight: bold; border: 1px solid {{object.user_id.company_id.email_secondary_color or '#875A7B'}}; border-bottom-start-radius: 3px;">
-                                        <t t-if="not object.allday">
-                                            <div>
-                                                <t t-out="format_time(time=object.start, tz=mail_tz, time_format='short', lang_code=object.env.lang)">11:00 AM</t>
-                                            </div>
-                                            <t t-if="mail_tz">
-                                                <div style="font-size: 10px; font-weight: normal">
-                                                    (<t t-out="mail_tz"> Europe/Brussels</t>)
-                                                </div>
-                                            </t>
+                        <h1 style="font-size: 18px; font-weight: bold; color: #374151; margin-bottom: 28px;">
+                            Event canceled
+                            <img src="/calendar/static/src/img/times-circle.png" style="display: inline-block; border:0; outline:none; width: 24px; height: 24px; margin-left: 10px; text-decoration:none; vertical-align: bottom;" />
+                        </h1>
+                        <p style="margin: 0;">
+                            Hello,<br/>
+                            <span style="display: block; margin-top: 8px;">
+                                This is to inform you that the event <strong><t t-out="event_name"/></strong>
+                                <t t-if="event_organizer">organized by </t><strong><t t-out="event_organizer"/></strong>
```

Comment:
Nice :) we can probably add placeholder values (used in template edition) like here for instance:
`The date of the meeting <strong t-out="object.event_id.name or ''">Follow-up for Project proposal</strong> <t t-if="organizer">created by </t><t t-out="object.event_id.user_id.partner_id.name or ''">Colleen Diaz</t> is now scheduled for`